src/helm/benchmark/window_services/wider_openai_window_service.py

```diff
@@ -10,7 +10,27 @@ def max_sequence_length(self) -> int:
         return 4000
 
 
-class GPT4WindowService(GPT2WindowService):
+class OpenAIChatWindowService(WiderOpenAIWindowService):
+    @property
+    def tokenizer_name(self) -> str:
+        return "openai/cl100k_base"
+
+    @property
+    def prefix_token(self) -> str:
+        """The prefix token"""
+        return "<|fim_prefix|>"
+
+
+class GPT3Dot5TurboWindowService(OpenAIChatWindowService):
```

Comment:
nit: let's name this `GPT3Point5TurboWindowService`?

Reply:
Done.